api/helpers/gamestate/publish-game-state.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
 
       const lastEventTargetType = () => {
         if (!['targetedOneOff', 'sevenTargetedOneOff'].includes(gameState.moveType)) {
-          return '';
+          return null;
```

Comment:
Switched targetType to be null when missing instead of an empty string. This way we will more consistently use `null` for missing/unused data in GameStates, GameStateRows, and PublishedGames